infra/core/ai/cognitiveservices.bicep

```diff
@@ -20,6 +20,9 @@ resource account 'Microsoft.CognitiveServices/accounts@2023-05-01' = {
     publicNetworkAccess: publicNetworkAccess
```

Comment:
Can you do a PR to update this too? https://github.com/Azure/azure-dev/blob/main/templates/common/infra/bicep/core/ai/cognitiveservices.bicep

Reply:
What updates are required? Do you just mean to pull from main?